src/scripts/safari-enable-automation.sh

```diff
@@ -13,22 +13,46 @@
 
 set -e
 
-safaridriver_binary="/Applications/Safari Technology Preview.app/Contents/MacOS/safaridriver"
+if [ $1 == "stable" ]; then
+  browser_name="Safari"
+  safaridriver_binary="/usr/bin/safaridriver"
+else
+  browser_name="Safari Technology Preview"
+  safaridriver_binary="/Applications/Safari Technology Preview.app/Contents/MacOS/safaridriver"
+fi
 
+# Create a session using a body which satisfies both the Selenium JSON Wire
+# protocol (for Safari 11.1 and earlier) and the W3C WebDriver protocol (for
```

Comment:
If we drop the Safari 11.1 support, is any simplification made possible? I guess that since you've already made it work, that's not really necessary though.

Reply:
Yeah, the timing of these things may have obviated this workaround. Still, I'd like to keep it here. First: it allows us to revert to 11 if that's ever necessary. Second: it's become like a red badge of courage for me.